lib/utils/encoding/sdkSuffix/getSDKSuffix.js

```diff
@@ -0,0 +1,52 @@
+let fs = require('fs');
+let path = require('path');
+let encodeVersion = require('./encodeVersion');
+
+let nodeSDKValue = 'M'; // Constant per SDK
+
+
+/**
+ * @description Turns x.y.z into x.y
+ * @param {string} semVerStr
+ */
+function removePatchFromSemver(semVerStr) {
+  let parts = semVerStr.split('.');
+
+  return `${parts[0]}.${parts[1]}`;
+}
+
+/**
+ * @description Gets the SDK signature by encoding the SDK version and node version
+ * @param {{isResponsive:boolean}} features
+ * @param {'default' | 'x.y.z' | 'x.y' | string} useSDKVersion Default uses package.json version
+ * @param {'default' | 'x.y.z' | 'x.y' | string} useNodeVersion Default uses process.versions.node
+ * @return {string} encodedSDK Suffix
+ */
+function getSDKSuffix(features = {}, useSDKVersion = 'default', useNodeVersion = 'default') {
+  try {
+    // allow to pass a custom SDKVersion
+    let pkgJSONFile = fs.readFileSync(path.join(__dirname, '../../../../package.json'), 'utf-8');
```

Comment:
no process.env.{something} to get this?

Reply:
Well if you think about it, we're a library that someone installs.
Even if there was a process.env.{something}, it would probably take the version of the installing project, and not our package.json.